### ISSUE TITLE
Feature Request: make getFrameHtmlResponse populate og metadata

### ISSUE THREAD

**Issue by spennyp** [coinbase/onchainkit#103]
### Describe the solution you'd like

Instead of using nextjs pages and `<FrameMetadata />` for the first frame, it can be useful to use a route, and a `GET` request. This minimizes the number of `page.tsx` needed for a frame project. 

For example what we are doing here: https://github.com/papercliplabs/frames/blob/main/src/app/auction/%5Bdao%5D/route.tsx#L6

But, in order for the image to load from the first `GET` request in a frame, `og:image` and `og:title` must be provided in the metadata. 

To solve this, it would be great if `getFrameHtmlResponse` included the `og:image`, and `og:title` metadata.
For example like we did here: https://github.com/papercliplabs/frames/blob/main/src/utils/metadata.tsx#L22

---

Another minor nit in comparing `getFrameHtmlResponse` with our `generateFrameMetadata` function is formatting of the metadata output

onchainkit's `getFrameHtmlResponse` output:
```html
<!DOCTYPE html><html><head><meta property="fc:frame" content="vNext" /><meta property="fc:frame:image" content="https://rnznv-2600-1700-7be1-630-9143-9b9f-ae38-fdd.a.free.pinggy.link/images/nouns-auction-house.png" /><meta property="fc:frame:button:1" content="View auction!" /><meta property="fc:frame:button:1:action" content="post" /><meta property="fc:frame:post_url" content="https://rnznv-2600-1700-7be1-630-9143-9b9f-ae38-fdd.a.free.pinggy.link/auction/nouns" /></head></html>
```

our `generateFrameMetadata` output:
```html
<!DOCTYPE html>
<html>
    <head>
	<meta property="og:image" content="https://frames-rouge.vercel.app/images/nouns-auction-house.png" />
	<meta property="og:title" content="Nouns DAO" />
	<meta property="og:description" content="One Noun everyday, forever!" />
	<meta property="fc:frame" content="vNext" />
	<meta property="fc:frame:image" content="https://frames-rouge.vercel.app/images/nouns-auction-house.png" />
	<meta property="fc:frame:post_url" content="https://frames-rouge.vercel.app/auction/nouns" />
	<meta property="fc:frame:button:1" content="View auction!" />
	<meta property="fc:frame:button:1:action" content="post" />

    </head>
</html>
    
```

The latter is much easier to read and debug in the browser!

**Comment by spennyp** [comment]
See it is also mentioned in this issue: https://github.com/coinbase/onchainkit/issues/44

**Comment by Zizzamia** [comment]
Sweet, let me take care this today!

**Comment by Zizzamia** [comment]
@spennyp but there is an argument to be made, that the flatten part should be optional, and let the developer pick when do it 🤔

**Comment by spennyp** [comment]
Ah yea fair, I'd say don't worry about the output part!

**Comment by Zizzamia** [comment]
Solved with `v0.6.1`.

Ready for your next wish :)
